### ISSUE TITLE
Modify this code for better accuracy

### ISSUE THREAD

**Issue by 1033020837** [lonePatient/Bert-Multi-Label-Text-Classification#40]
According to this document:https://huggingface.co/transformers/model_doc/bert.html:
the pooler_output of bert_model is usually not a good summary of the semantic content of the input, we should better with averaging or pooling the sequence of hidden-states for the whole input sequence.

So modify the BertForMultiLabel.py to:
avg_output = torch.mean(outputs[0],1).view(-1,self.config.hidden_size)
logits = self.classifier(avg_output)

You could get better performance through this modification.

**Comment by lonePatient** [comment]
@1033020837 Thanks a lot, I will try.
